.github/workflows/build.yml

```diff
@@ -19,3 +19,10 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+      - name: Check code is formatted
+        run: |
+          if [ "$(gofmt -s -l . | wc -l)" -gt 0 ]; then
+            echo "Found code needing formating, please run gofmt -s -w before creating the PR"
```

Comment:
@cristiGuranIonos not sure if it matters but when we are doing the check in the if condition, we are using `-l` option, but, in the displayed message, we are asking the user to run `go fmt` using `'-s` and `-w` options. I just want to make sure that we use proper options both in check / message displayed to the user.

Reply:
@cristiGuranIonos explained me: `-l` option is used for listing, `-w` option is used for formatting, so it is correct.